### PR TITLE
docs: Updated migration docs for v13 release

### PR DESCRIPTION
**Pull request by mmulji-ic**
## Description

Part of epic https://github.com/cosmos/gaia/issues/2741

Updated docs for v13 release.

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct `docs:` prefix in the PR title
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] Confirmed the correct `docs:` prefix in the PR title
- [ ] Confirmed all author checklist items have been addressed 
- [ ] Confirmed that this PR only changes documentation
- [ ] Reviewed content for consistency
- [ ] Reviewed content for thoroughness
- [ ] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)

### CODE REVIEW

**Review by mpoke — APPROVED** [review]
LGTM

@mmulji-ic could we move from keeping historical upgrading docs, as they're all quite similar and instead use `./UPGRADING.md` when cutting a new release?

File: docs/migration/cosmoshub-4-v13-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+#### Expected result
+
+When the upgrade block height is reached, you can find the following information in the log:
+
+```shell
+ERR UPGRADE "v13" NEEDED at height: 17380000: upgrade to v13 and applying upgrade "v13" at height:17380000
+```
+
+Then the Cosmovisor will create `$GAIA_HOME/cosmovisor/upgrades/v13/bin` and download the Gaia v13.0.0 binary to this folder according to links in the `--info` field of the upgrade proposal.
+This may take 7 minutes to a few hours, afterwards, the chain will continue to produce blocks once validators with a total sum voting power > 2/3 complete their nodes upgrades.
```
Comment (yaruwangway) [inline docs/migration/cosmoshub-4-v13-upgrade.md:240]: 7 min is very specific, maybe change to a few mins. 7 min is from estimation of the testnet of theta upgrade.

**Comment by yaruwangway** [comment]
The cosmovisor tutorial might be slightly different now.

**Comment by mmulji-ic** [comment]
> LGTM
> 
> @mmulji-ic could we move from keeping historical upgrading docs, as they're all quite similar and instead use `./UPGRADING.md` when cutting a new release?

I want to start updating content remove duplication after the docusaurus migration.

**Review by sainoe — APPROVED** [review]

**Comment by sainoe** [comment]
LGTM!

**Comment by mmulji-ic** [comment]
> The cosmovisor tutorial might be slightly different now.

We used effectively the same tutorial for the last upgrade, so hopefully its also good this time around. For future releases we want to streamline this totally, as theres a lot of repeated material and its easy for maintainers to make mistakes.

**Pull request merged**